Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettingsSystemComponent.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/Entity.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzToolsFramework/PaintBrushSettings/PaintBrushSettingsRequestBus.h>
+
+namespace AzToolsFramework
+{
+    //! PaintBrushSettings defines the specific global paintbrush settings.
+    //! They can be modified directly through the Property Editor or indirectly via the PaintBrushSettingsRequestBus.
+    class PaintBrushSettings
+    {
+    public:
+        AZ_CLASS_ALLOCATOR(PaintBrushSettings, AZ::SystemAllocator, 0);
+        AZ_RTTI(PaintBrushSettings, "{CE5EFFE2-14E5-4A9F-9B0F-695F66744A50}");
+        static void Reflect(AZ::ReflectContext* context);
+
+        virtual ~PaintBrushSettings() = default;
```

Comment:
question: Do we expect this class to be overridden in future? Just want to check as we might be able to use AZ_TYPE_INFO instead and get rid of the virtual destructor